test/framework/clientset.go

```diff
@@ -15,7 +15,12 @@ import (
 	//clientmachineconfigv1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
 )
 
+type Config struct {
+	Namespace string `envconfig:"NAMESPACE" default:"openshift-special-resource-operator"`
+}
+
 type ClientSet struct {
+	Config Config
```

Comment:
I like the `Config` struct, but I am wondering if it should belong to `ClientSet` at all.

Reply:
I wondered the same thing and eventually though it can be, the way I think of it as sort of a "context" for the client.

The only other option I though about is to just pass `Config` to each one of the methods that somewhere in the method tree uses it - I preferred the first option.

Reply:
Let's go with this for now.